src/routes.ts

```diff
@@ -1,30 +1,33 @@
+
+const baseUrl = `api/v2/storefront`;
```

Comment:
Can we use `storefrontUrl` instead of `baseUrl`, please? Will be more descriptive.

Reply:
Actually, storefrontPath would be even more suitable, but I changed it to `storefrontUrl` now.

Reply:
You're right, let's use `storefrontPath`.